src/storage.js

```diff
@@ -0,0 +1,34 @@
+//get config
+require('./config');
+//get redis constant searches redis globally 
+
+const Redis = require('ioredis'); //chnage to ioredis
+//connecting client
+const client = new Redis(process.env.REDIS_URL);
+
+//result from require, displays externally
+module.exports = {
+    addFeed: function(name, url, f) {
+      const feed = { 'name': name, 'url': url }; //insted of json store id to a hash
+      client.pipeline().incr('user_id').get('user_id', function(err, id) {
+          if (err)
+            f(err, id);
+          else
+            client.hset('feeds', id, JSON.stringify(feed), f); //transform json object into string and save to redis feeds
```

Comment:
Why not use `hmset` and then you can pass an Object vs. having to stringify to JSON?

Reply:
See https://github.com/luin/ioredis/blob/90b1aecb4cc9f340aacb0dd54b200819092a4597/test/functional/transaction.ts#L136-L156 for an example of this.

Reply:
I have not considered using hmset because it is deprecated as stated here. https://redis.io/commands/hmset

However, upon your recommendation I have tried it and here is the result it was giving me. 
![image](https://user-images.githubusercontent.com/28875884/68962831-7871a980-07a3-11ea-85cb-fc1ace408b4e.png)

Therefore I will stick with stringify

Reply:
OK, `hset` it is.  But I still don't understand why you don't just set multiple key/value pairs as specified in https://redis.io/commands/hset.  Adding extra cost of stringify/parse seems unnecessary if it will do it for us automatically.